helpers/mappers.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+    intToBoolean,
+    booleanToint,
+    userToBody,
+    adventureToBody, // s 
+  };
+  
+  function intToBoolean(int) {
+    return int === 1 ? true : false;
```

Comment:
Since you're just already using a boolean condition here, you don't need the ternary:
```javascript
function intToBoolean(int) {
  return int === 1;
}
```